pod/annotations.go

```diff
@@ -206,6 +206,11 @@ const (
 	AnnotationKeyPredRuntimeQuantiles    = "runtime.predictions.titus.netflix.com/quantiles"
 	AnnotationKeyPredRuntimeModelVersion = "runtime.predictions.titus.netflix.com/model-version"
 	AnnotationKeyPredRuntimeModelID      = "runtime.predictions.titus.netflix.com/model-id"
+
+	// mockPod
+	AnnotationKeyPodParameterMockPodPrepareTime = "mockPod.netflix.com/killTime"
```

Comment:
I  think we need 1 boolean here to indicate that a pod is a mockPod.

Otherwise we need to infer that from the presence of at least 1 of these fields, but I think that is less explicit.

Reply:
We can add a function that would abstract it: IsMockPod().. 'runTime' seems like the required annotation so could use the presence of that.

Reply:
As long as that will really be a required annotation.
Heck even if it changes, the helper function is the important thing.